manuscript/draft.Rmd

```diff
@@ -253,71 +252,60 @@ The site-observer model found that 70% of the variance in richness in the
-for forecasting within-site variability was either the "average" baseline model
-or the ARIMA model [[Not sure what the bit about within-site variability means
-here. Also, didn't Naive usually win when observer effects weren't corrected
-for?]] (Figure 3). In general, each model's error increased more-or-less
-linearly with time, although the slopes and initial error rates varied across
-models (Figure 3). The site-level models (i.e. the baselines and the ARIMA
-methods) tended to do very well (Figure 3) on short time horizons, especially in
-terms of RMSE.
-
-In general, the models produced confidence intervals that were too narrow,
-indicating overconfident predictions. Of these, the random forest-based SDM
-stack was the most overconfident, with observed richness values frequently
-falling well outside their confidence limits (Figure 3C). As noted elsewhere
-[@harris2015], this overconfidence is a product of the assumption in stacked
-SDMs that errors in the species-level predictions are independent. The JSDM and
-macroecological model has the best coverage, though each model still
-underestimated the possible variation in richness. The "naive" model
-was the only model whose confidence intervals were too wide (Figure 3), which
-can be attributed to the rapid rate at which these intervals expand as the time
-lag increases (Figure 1).
-
-RMSE and deviance produced qualitatively similar results in most cases, with the
-notable exception of the stacked SDMs (Figure 3A and 3B). Deviance, unlike RMSE,
-depends on the full predictive distribution of a model, including the model's
-estimated uncertainty. As a result, the stacked SDMs' deviance was notably
-higher than the next-worst model even though its RMSE was not unusual. On the
-other hand, the GBM-based "macroecological" model and especially the mistnet
-JSDM had reasonably well-calibrated uncertainty estimates, even in the presence
-of uncorrected observer biases (Figure 3C). This prevented their deviance from
-rising as quickly as the site-level models as the observer population evolved
-and resulted in better comparative performance for these models in terms of
-deviance (Figure 3B).
+(Figure 2; R^2 XX-XX%). However SDMs (both stacked and joint) and the
+macroecological model all failed to successfully forecast the highest-richness
+sites resulting in a notable clustering of predicted values near ~60 species and
+the poorest performance in the cross-site comparison (R^2 XX-XX%).
+
+In general, each model's error increased more-or-less linearly with time,
+although the slopes and initial error rates varied across models (Figure 3). The
+site-level models (i.e. the baselines and the ARIMA methods) tended to do very
+well (Figure 3), especially on short time horizons, and especially in terms of
+RMSE. Of these, the "Naive" model performed especially well at sites with
+uncorrected observer turnover (since its predictions were best-calibrated to
+observers that were active in later years).
```

Comment:
Two thoughts about the revision of this paragarph:

1. I still don't personally see the value of focusing on the performance of the uncorrected model when discussing general overall performance. Can you walk me through you thinking on why it's useful to include here rather than saving those results for the paragraph on the observer model? Also, since we decided not to include non-observer model results in Figures 3 and 4 we'd need to jump to citing 5 here to bring this up at this point. That's OK if we need to, but I think that's part of my general confusion over the value of talking about this here.

2. Since we just finished a paragraph talking generally (i.e., not with respect to time lag) about the among sites comparison it seems weird to then move to the within site dynamics and then not talk about them generally before delving into the changes with time horizon. That's why I added the opening sentence speaking generally about the results, to help with the transition and provide a high level summary of the results.

If we end up adding some version of the current opening sentence back in I think we should probably plan to flip the order on Figures 3 and 4, so that the violin plots come first and get cited in the first sentence and the temporal trends come second.